frontend/src/lib/components/Papercups.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
```

Comment:
@joethreepwood what is up with your local branch 😆

Reply:
I wish I knew!